src/getCommitMeaning.test.ts

```diff
@@ -40,6 +40,27 @@ describe("getCommitMeaning", () => {
 		["chore(deps): release", "release"],
 		["chore(deps): release 1.2.3", "release"],
 		["chore(deps): release v1.2.3", "release"],
+		["chore!: message", "meaningful"],
+		["docs!: message", "meaningful"],
+		["chore!: release", "meaningful"],
+		["feat!: a feature with a breaking change", "meaningful"],
+		["chore: bump\n\nBREAKING CHANGE: breaks things", "meaningful"],
+		["chore: bump\n\nBREAKING-CHANGE: breaks things", "meaningful"],
+		["docs: bump\n\nBREAKING CHANGE: breaks things", "meaningful"],
+		["docs: bump\n\nBREAKING-CHANGE: breaks things", "meaningful"],
+		[
+			"chore: deps\n\nBREAKING-CHANGE: breaks things\nMultiple: footer notes",
+			"meaningful",
+		],
+		["chore: deps\n\n! in the commit body", { type: "chore" }],
+		[
+			"chore: deps\n\nFooter-note: other than BREAKING CHANGE",
+			{ type: "chore" },
+		],
+		[
+			"chore: deps\n\nMultiple: footer notes\nMultiple: footer notes",
+			{ type: "chore" },
+		],
```

Comment:
[Testing] Thinking on the details, there's some nuance around the `BREAKING CHANGE:` note that I think should be captured in tests:

* What if a line starts with `BREAKING CHANGE` (no `:`)?
* What if a line contains `BREAKING CHANGE: ` inside it, but not at the start?
* What if a line starts with something like `BREAKING CHANGE (major):`? Should that be considered meaningful? _(I'm guessing not)_

The answers to these should be in the unit tests.